### ISSUE TITLE
Adding formats in preferences inaccessible to Voiceover users

### ISSUE THREAD

**Issue by Quartizer** [GetiPlayerAutomator/get-iplayer-automator#542]
Using Voiceover screen reader I am unable to add formats in the BBC formats tab of preferences, I assume this is the same for ITV as well, but have had no need to try this. I can press the button to add another row to the table, but to VO this appears blank, and there is no accessible way to arrow through the list of available options, which I'm reliably assured by my sighted partner, appears.

Thanks, I look forward to your response.

**Comment by dinkypumpkin** [comment]
I have posted a pre-release build with a fix for this issue (direct link below), if you wish to try it out. VoiceOver will now identify format preference entries as popup buttons with default value of "Please select...". When popup button is invoked, its associated menu will appear and items will be spoken as you arrow through them.

General info for pre-release builds:
https://sourceforge.net/projects/getiplayerauto/files/prerelease/

Direct link:
https://sourceforge.net/projects/getiplayerauto/files/prerelease/1.8.7-pre.2/GetiPlayerAutomator_1.8.7-pre.2.zip/download

**Comment by Quartizer** [comment]
This is awesome and very much appreciated, thanks. :)
